oeps/processes/oep-0054-core-contributors.rst

```diff
@@ -269,10 +269,16 @@ commit rights generally holds commit rights for only some of the 150+
 
 Expanding a CC's responsibilities should follow the same process as establishing
 a new contributor, although in most cases, the comment period need only be one
-week. However, in cases where the new responsibilities are greatly different
+week. In addition, for newly created repositories or similar situations, a
+single forum thread can be used to nominate several CCs at once, provided they
```

Comment:
```suggestion
single forum thread can be used to nominate several existing CCs at once, provided they
```
I think it may be clearer to put "existing" here. Take it or leave it :)